### DISCUSSION TITLE
Game Stick Lite 4k

### DISCUSSION THREAD

**Discussion by horsiezoo** [nk64/GT4286#15] [General]
Under ./ui folder there's a .png file logo--.png "Game Stick Lite 4k"

Googling brings up : 

https://4pda.to/forum/index.php?showtopic=1035112&st=14840 (Russian)

Some of which seems similar to this console, they have got as far as making a CFW for that system : 

https://nnmclub.to/forum/viewtopic.php?t=1586865

**Comment by nk64** [comment]
I saw that and went a little way down that path but I gave up (maybe too quickly), thinking that maybe this was once just marketed as a cheap clone of a Game Stick Lite 4k (without actually being similar hardware).

Let me know if you have any luck signing up and downloading the torrent on your second link.
[NNMClub.to]_GameStickLite_stick-ow.pro_OpenWorld_v090plus.7z.torrent

**Reply by horsiezoo** [reply]
magnet:?xt=urn:btih:9E976588A197D17C60BD59EFD383B5B43CEF4CAA
No sign up needed, dl'd / extracted, flashed to an sd, no boot, there are a few boot files for different models of GS lite and its quite specific to the hardware so .. further exploration needed. I'm gathering GS lite puts everything including OS on sd card so works a little differently.

**Reply by nk64** [reply]
Must have been my old torrent client, the magnet link was just hanging. Upgraded my client and dowloading now.

**Comment by Hello-dot-Jpg** [comment]
![image](https://github.com/user-attachments/assets/e8acc2c0-fbda-49b5-a9a2-7962c26f51ea)

Unfortunately, these seem to use other (better?) hardware, they also have a dedicated GPU, I suspect our "Game Stick 4k" is a lower cost copy

**Reply by horsiezoo** [reply]
Hmm yes I read that, if you go back on the forum under “other consoles” there’s a whole lot more stuff to trawl through
